packages/components/src/image/docs/Image.stories.mdx

```diff
@@ -18,9 +18,13 @@ import { Inline } from "@components/layout";
     githubPath="/packages/components/src/image/src"
 />
 
-## Accessibility
+## Guidelines
 
-An image must have an `alt` attribute.
+### Accessibility
+
+- An image must have an `alt` attribute.
```

Comment:
While this is true I think it's important to specify that the `alt` attribute **can** be empty.
It is written in the documentation linked bellow but we know everyone's not gonna click it
> If the image doesn't require a fallback (such as for an image which is decorative or an advisory icon of minimal importance), you may specify an empty string ("").

Reply:
With the new branding, we use mostly SVG illustration so it's not so much the case anymore, but in most apps we build images are used in a decorative way and should use an empty `alt`. Some examples I found:
`Image` components used
![image](https://user-images.githubusercontent.com/234418/166475676-55508738-b5a5-4a15-8272-93583d526791.png)
![image](https://user-images.githubusercontent.com/234418/166475867-fa84c38b-383f-401a-abb5-9cd345899567.png)
`SvgImage` components used
![image](https://user-images.githubusercontent.com/234418/166476117-a73edf5d-9337-47b4-ab10-662a86fcd48c.png)
![image](https://user-images.githubusercontent.com/234418/166476239-ef2af03c-234c-44ba-aef7-0c97c776a3fa.png)

Reply:
Make sense then!